hw/ip/flash_ctrl/data/flash_ctrl_testplan.hjson

```diff
@@ -24,116 +24,112 @@
       tests: ["flash_ctrl_smoke"]
     }
     {
-      name: backpressure
+      name: smoke_sw_op
       desc: '''
-            Sanity + randomly pick between back-pressured reads / writes and fifo status-polled
-            reads / writes via the controller.
+            Perform flash protocol controller read, program and erase on the single page of one 
+            bank within Data partition.
```

Comment:
this might be covered later, but will we also cover them on non-data partitions?

Reply:
Testplan was written in a manner that first simple tests are being developed together with environment. Later on, tests will be more complex and with more randomization covering more features.